tests/e2e/test_api.py

```diff
@@ -804,42 +805,50 @@ def test_feedback_storing_standalone():
-#     pod_name = cluster_utils.get_single_existing_pod_name()
-
-#     # there are multiple tests running agains cluster, so transcripts
-#     # can be already present - we need to ensure the storage is empty
-#     # for this test
-#     feedbacks = cluster_utils.list_path(pod_name, feedbacks_path)
-#     if feedbacks:
-#         cluster_utils.remove_dir(pod_name, feedbacks_path)
-#         assert cluster_utils.list_path(pod_name, feedbacks_path) == []
-
-#     response = client.post(
-#         "/v1/feedback",
-#         json={
-#             "conversation_id": CONVERSATION_ID,
-#             "user_question": "what is OCP4?",
-#             "llm_response": "Openshift 4 is ...",
-#             "sentiment": 1,
-#         },
-#         timeout=BASIC_ENDPOINTS_TIMEOUT,
-#     )
-
-#     assert response.status_code == requests.codes.ok
-
-#     feedback_data = cluster_utils.get_single_existing_feedback(pod_name, feedbacks_path)
-
-#     assert feedback_data["user_id"]  # we don't care about actual value
-#     assert feedback_data["conversation_id"] == CONVERSATION_ID
-#     assert feedback_data["user_question"] == "what is OCP4?"
-#     assert feedback_data["llm_response"] == "Openshift 4 is ..."
-#     assert feedback_data["sentiment"] == 1
+@pytest.mark.cluster()
+def test_feedback_storing_cluster():
+    """Test if the feedbacks are stored properly."""
+    # constant from tests/config/cluster_install/ols_manifests.yaml
+    feedbacks_path = OLS_USER_DATA_PATH + "/feedback"
+    pod_name = cluster_utils.get_single_existing_pod_name()
+
+    # disable collector script to avoid interference with the test
+    cluster_utils.create_file(pod_name, OLS_COLLECTOR_DISABLING_FILE, "")
+
+    # there are multiple tests running agains cluster, so transcripts
+    # can be already present - we need to ensure the storage is empty
+    # for this test
+    feedbacks = cluster_utils.list_path(pod_name, feedbacks_path)
+    if feedbacks:
+        cluster_utils.remove_dir(pod_name, feedbacks_path)
+        assert cluster_utils.list_path(pod_name, feedbacks_path) == []
+
+    response = client.post(
+        "/v1/feedback",
+        json={
+            "conversation_id": CONVERSATION_ID,
+            "user_question": "what is OCP4?",
+            "llm_response": "Openshift 4 is ...",
+            "sentiment": 1,
+        },
+        timeout=BASIC_ENDPOINTS_TIMEOUT,
+    )
+
+    assert response.status_code == requests.codes.ok
+
+    feedback_data = cluster_utils.get_single_existing_feedback(pod_name, feedbacks_path)
+
+    assert feedback_data["user_id"]  # we don't care about actual value
+    assert feedback_data["conversation_id"] == CONVERSATION_ID
+    assert feedback_data["user_question"] == "what is OCP4?"
+    assert feedback_data["llm_response"] == "Openshift 4 is ..."
+    assert feedback_data["sentiment"] == 1
+
+    # ensure script is enabled again after successful test
+    cluster_utils.remove_file(pod_name, OLS_COLLECTOR_DISABLING_FILE)
```

Comment:
you need to do this in a finally block because otherwise if any of the asserts fail, this won't get executed.

https://stackoverflow.com/questions/56938028/execute-code-after-an-assert-in-case-of-assertionerror